awx/main/management/commands/host_metrics.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand
+from dateutil import parser
+from django.utils.timezone import now
+import pytz
+from datetime import datetime, timezone
+from awx.main.models.inventory import HostMetrics
+import sdb
+
+
+class Command(BaseCommand):
+
+    help = 'This is for offline licensing usage'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--since', dest='since', action='store', help='Start date for data')
+        parser.add_argument('--until', dest='until', action='store', help='End date for data')
+
+    def handle(self, *args, **options):
+        opt_since = options.get('since')
+        opt_until = options.get('until')
+        if opt_since:
+            since = parser.parse(opt_since)
+            since = since.replace(tzinfo=timezone.utc)
```

Comment:
I had to add ```tzinfo=timezone.utc``` to make the time timezone aware. The ```datetime``` field in postgres SQL is time aware.

Reply:
I wouldn't do this unconditionally.  A string that gets parsed by `parser.parse` might have timezone information encoded in it, in which case the resulting datetime object will be timezone aware.  Blowing that away to become UTC would be a problem then in that case.

Reply:
something like, `if since.tzinfo is None:`